### PR TITLE
Update cleanup script

### PR DESCRIPTION
**Pull request by manju956**
Source paths of many modules in the cleanup script are updated to
the latest. This script will be useful in cleaning up build artifacts
in non-docker builds.

Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

File: tools/clean.sh [hunk 1]
```diff
@@ -27,31 +27,48 @@ SRCDIR="$(realpath ${SCRIPTDIR}/..)"
...
-cd $SRCDIR/common/python
-make clean
+# --------------- COMMON CPP ---------------
+cd $SRCDIR/common/cpp
+rm -rf build
 
-# --------------- ENCLAVE MANAGER ---------------
-cd $SRCDIR/enclave_manager
-SGX_MODE=${SGX_MODE:-SIM} make clean
+# --------------- SGX WORKLOAD ---------------
```
Comment (danintel) [inline tools/clean.sh]: change SGX to INTEL SGX

File: tools/clean.sh [hunk 2]
```diff
@@ -27,31 +27,48 @@ SRCDIR="$(realpath ${SCRIPTDIR}/..)"
...
+cd $SRCDIR/common/sgx_workload
+rm -rf build
 
-# --------------- COMMON SGX WORKLOAD ---------------
-rm -rf $SRCDIR/common/sgx_workload/build
+# --------------- SGX COMMON ---------------
```
Comment (danintel) [inline tools/clean.sh]: change SGX COMMON to INTEL SGX COMMON

File: tools/clean.sh (hunk 1 above)
Comment (manju956) [reply tools/clean.sh]: Done

File: tools/clean.sh (hunk 2 above)
Comment (manju956) [reply tools/clean.sh]: Done

**Pull request merged**